application/Models/Calculator.php

```diff
@@ -106,4 +106,32 @@ public function divide()
 
         return (float) $this->firstNumber / $this->secondNumber;
     }
+
+    /**
+     * Return square of $firstNumber in Calculator class
+     * return float|int
+     */
+    public function square()
+    {
+        return $this->firstNumber * $this->firstNumber;
+    }
+
+    /**
+     * Return cube of $firstNumber in Calculator class
+     * return float|int
+     */
+    public function cube()
+    {
+        return $this->firstNumber *  $this->firstNumber * $this->firstNumber;

```

Comment:
aqui podrias usar la funcion pow de php.

```suggestion
        return pow($this->firstNumber, 3);
```